### PR TITLE
Added Attribute Aggregation as Corto inbound filter

### PR DESCRIPTION
**Pull request by oharsta**
Feature branch for attribute aggregation in EB for 5.x branch.

### CODE REVIEW

**Comment by DRvanR** [comment]
Note: requires https://github.com/OpenConext/OpenConext-deploy/tree/engineblock-5.x-dev to be up to date with master before being able to merge.

File: application/configs/application.ini [hunk 1]
```diff
@@ -209,6 +209,17 @@ pdp.username = "pdp_admin"
 pdp.password = "secret"
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;; ATTRIBUTE AGGREGATION SETTINGS ;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+; Location of AA
+attributeAggregation.baseUrl = "https://aa.surfconext.nl/aa/api/attribute/aggregate"
```
Comment (DRvanR) [inline application/configs/application.ini]: Is this a wanted default? Can imagine that this should be something not-surfconext?

Comment (thijskh) [inline application/configs/application.ini]: `aa.demo.openconext.org` seems to follow what has been done in other places. Although we'd have to admit that many other of these defaults in application.ini currently suffer from the same problem.

Comment (DRvanR) [inline application/configs/application.ini]: Personally I'd opt for `aa.vm.openconext.org` to ensure that it works with the basic local install using OpenConext-deploy.

Comment (DRvanR) [inline application/configs/application.ini]: The usage of SURFconext is indeed quite prolific, this will be fixed in a separate PR.
